src/dmd/root/stringtable.d

```diff
@@ -126,7 +123,7 @@ public:
     Returns: the string's associated value, or `null` if the string doesn't
      exist in the string table
     */
-    inout(StringValue)* lookup(const(char)[] str) inout nothrow pure @nogc
+    inout(StringValue)* lookup(const(char)[] str) inout @nogc nothrow pure
```

Comment:
This change adds nothing.

Reply:
It makes it adhere to D-style

Reply:
I find little value in such changes, but if other folks want it...